### PR TITLE
Completed CSS Flexbox

### PR DESCRIPTION
**Pull request by macyhongg**
## Justify Content ##
image

## Align Items ##
image

## Flex Wrap ##
image

## Display Flex ##
image

### CODE REVIEW

File: css-flexbox/justify-content.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: 25%;
```
Comment (BrettAlbright) [inline css-flexbox/justify-content.html]: flex basis is a flex item property, so make sure it is on the wrapper itself.

File: css-flexbox/align-items.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100%/3);
```
Comment (BrettAlbright) [inline css-flexbox/align-items.html]: this should be on the card wrapper since its a flex item property

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates are needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

File: css-flexbox/align-items.html (hunk 2 above)
Comment (macyhongg) [reply css-flexbox/align-items.html]:
```
    .card-wrapper {
    flex-basis: calc(100%/3);
    }
```

**Pull request merged**